imblearn/utils/estimator_checks.py

```diff
@@ -12,20 +12,25 @@
 from collections import Counter
 
 import numpy as np
+from scipy import sparse
 
 from sklearn.datasets import make_classification
+from sklearn.cluster import KMeans
 from sklearn.utils.estimator_checks import _yield_all_checks \
     as sklearn_yield_all_checks, check_estimator \
     as sklearn_check_estimator, check_parameters_default_constructible
 from sklearn.exceptions import NotFittedError
 from sklearn.utils.testing import (assert_warns, assert_raises_regex,
                                    assert_true, set_random_state,
-                                   assert_equal)
+                                   assert_equal, assert_allclose,
```

Comment:
I don't understand why but this PR would insert back the `assert_equal` and the diff looks wierd 'cos they are no longer there in master (see [this](https://github.com/scikit-learn-contrib/imbalanced-learn/blob/master/imblearn/utils/estimator_checks.py#L21))